packages/components/src/FilterBar/context/FilterBarContext.tsx

```diff
@@ -81,6 +82,10 @@ export const FilterBarProvider = <ValuesMap extends FiltersValues>({
       onValuesChange({ ...values, [id]: undefined })
     },
     getInactiveFilters: () => getInactiveFilters<ValuesMap>(state),
+    clearAllFilters: () => {
```

Comment:
Why not call it `deactivateFilters`?

Reply:
Because it also clears the values on top of deactivating :) It could be `clearFilters` if the `all` feels verbose?

Reply:
Perhaps the action does too much if it both clears and deactivates?

Reply:
Nah, it's the correct action for the UX. It's only being called separately because `values` are held outside of the context 😅

Reply:
I don't think the action needs to match the UX, we could have a `clearValues` and a `deactivateFilters` action , both of which are triggered in a single button. I think this makes the interactions easier to read and maintain eg. if we want to update how clearing works but not how deactivation works, it's easier to trace.

Reply:
Had a discussion, and I'm going to refactor the way the context gets the values. This should simplify feature implementation by having a single source of truth as opposed to two.